benchmarks/Benchmarks.Trace/Benchmarks.Trace.csproj

```diff
@@ -19,15 +19,23 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Datadog.Trace.BenchmarkDotNet\Datadog.Trace.BenchmarkDotNet.csproj" />
+    <ProjectReference Include="..\..\src\Datadog.Trace.ClrProfiler.Managed\Datadog.Trace.ClrProfiler.Managed.csproj" />
     <ProjectReference Include="..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Net.Http" />
```

Comment:
Should this be a `PackageReference` when targeting `netcoreapp3.1`?

Reply:
Does it? I never know what to do with that reference.

There's a warning at compilation time, maybe that's why

Reply:
I'm not sure either, but I always go with the "everything is a nuget package" mentality in .NET Core. I could be wrong!

Reply:
Switching to PackageReference fixes the warning